vs_utils/scripts/public_data/featurize_pdbbind.py

```diff
@@ -39,7 +39,7 @@ def featurize_pdbbind(pdbbind_dir, pickle_out):
   subdirs = [d for d in os.listdir(pdbbind_dir) if
       os.path.isdir(os.path.join(pdbbind_dir, d))]
   # TODO(rbharath): ONLY FOR DEBUGGING!
-  subdirs = subdirs[900:]
+  #subdirs = subdirs[900:]
```

Comment:
I think you can go ahead and just remove this line (and line 41 with the TODO as well).